libs/zedUpload/awsutil/s3api.go

```diff
@@ -18,8 +18,12 @@ import (
 )
 
 const (
-	S3_PART_SIZE        = 5 * 1024 * 1024
-	S3_PART_LEAVE_ERROR = true
+	//S3Concurrency parallels parts download limit
+	S3Concurrency = 5
+	//S3PartSize size of part to download
+	S3PartSize = 5 * 1024 * 1024
```

Comment:
Tiny nit -- but would it make more sense to say `S3PartSize = S3Concurrency * 1024 * 1024` or is not not implied?

Reply:
Well, they are separate. We run 5 routines, they all download their part